configure

```diff
@@ -115,6 +115,10 @@ if [[ ! -e "$SWIG_PATH" ]]; then
   echo "Can't find swig.  Ensure swig is in \$PATH or set \$SWIG_PATH."
   exit 1
 fi
+# Convert swig path to Windows style before writing into swig_path
+if is_windows; then
+  SWIG_PATH="$(cygpath -m "$SWIG_PATH")"
```

Comment:
Is there a reason to use `-m` ("mixed") rather than the more natural `-w` ("Windows names")?